llpc/util/llpcElfWriter.cpp

```diff
@@ -1167,21 +1378,40 @@ Result ElfWriter<Elf>::LinkGraphicsRelocatableElf(
 
     // Now get the symbols that belong in the symbol table.
     uint32_t offset = 0;
+    std::vector<RelocationEntry> relocations;
+
+    // Insert a dummy symbol. objdump somehow doesn't recognize the first entry in the symbol table section.
```

Comment:
Update the comment: This is a requirement of the ELF spec.

Reply:
Done.